src/Validator/AbstractValidatorChainEM2.php

```diff
@@ -48,7 +48,7 @@ public function __construct(StorageInterface $storage)
      * @param int $priority
      * @return \Zend\Stdlib\CallbackHandler
      */
-    public function attach($event, $callback = null, $priority = 1)
+    public function attach($event, callable $callback, $priority = 1)
```

Comment:
This is incorrect.

Yes, the argument cannot be null.

However, the argument _does not need to be callable_; `ValidatorChainTrait::attachValidator` can accept a callable or a `ValidatorInterface`.

Ideally, `ValidatorChainTrait::attachValidator()` should check for a non-callable `$callback` before calling `parent::attach()`.

All told, whatever fix is applied _needs a test_ so we can demonstrate what is being fixed, as well as ensure the fix is not accidentally removed in the future.